core/import-cobalt.c

```diff
@@ -73,20 +73,19 @@ extern int cobalt_buddies(void *handle, int columns, char **data, char **column)
  * Subsurface star rating.
  */
 
-extern int cobalt_visibility(void *handle, int columns, char **data, char **column)
+static int cobalt_visibility(void *handle, int columns, char **data, char **column)
 {
-	UNUSED(handle);
 	UNUSED(columns);
 	UNUSED(column);
 	UNUSED(data);
 	return 0;
 }
 
-extern int cobalt_location(void *handle, int columns, char **data, char **column)
+static int cobalt_location(void *param, int columns, char **data, char **column)
 {
-	UNUSED(handle);
 	UNUSED(columns);
 	UNUSED(column);
+	struct parser_state *state = (struct parser_state *)param;
 
 	static char *location = NULL;
```

Comment:
i guess it makes sense to be moved in the state as a generic "location" or possibly "cobalt_location" param.

Reply:
I added a second commit, which makes the code more explicit. But this is only compile tested! Do we have Cobalt files? I don't see any obvious things in the dives-directory.